xbmc/application/AppParams.h

```diff
@@ -50,10 +50,10 @@ class CAppParams
   const std::string& GetLogTarget() const { return m_logTarget; }
   void SetLogTarget(const std::string& logTarget) { m_logTarget = logTarget; }
 
-  std::string_view GetAudioBackend() const { return m_audioBackend; }
+  const std::string_view& GetAudioBackend() const { return m_audioBackend; }
```

Comment:
This returns a stack address and will crash. Same below.

Reply:
Are you sure about that? The address will point to a valid address during the lifetime of the parent class? What exactly will crash?

Reply:
`m_audioBackend` is a `std::string` which is converted to a temporary `std::string_view` and that local temporary is returned by reference.

Reply:
`const std::string_view&` this is bad to do
const its not needed with string_view since the underlayer data are already const,
the reference its not needed since the data its already a pointer...

Reply:
I'm gonna change it to `std::string&` which gets rid of the underlying copying